pkg/tinydtls/Makefile.include

```diff
@@ -63,9 +63,34 @@ endif
 PEER_MAX := $(or $(CONFIG_DTLS_PEER_MAX),$(patsubst -DCONFIG_DTLS_PEER_MAX=%,%,$(filter -DCONFIG_DTLS_PEER_MAX=%,$(CFLAGS))))
 ifneq (,$(PEER_MAX))
     CFLAGS += -DDTLS_PEER_MAX=$(PEER_MAX)
+else ifneq (,$(filter gcoap_dtls,$(USEMODULE)))
+    # The default value in sys/include/net/dtls.h for CONFIG_DTLS_PEER_MAX is 2
+    # when gcoap_dtls is active, otherwise 1. As the default in tinydtls is 1,
+    # we need to set it explicitly if the dtls.h default value deviates from
+    # the tinydtls default.
+    CFLAGS += -DDTLS_PEER_MAX=2
 endif
 
 HANDSHAKE_MAX := $(or $(CONFIG_DTLS_HANDSHAKE_MAX),$(patsubst -DCONFIG_DTLS_HANDSHAKE_MAX=%,%,$(filter -DCONFIG_DTLS_HANDSHAKE_MAX=%,$(CFLAGS))))
 ifneq (,$(HANDSHAKE_MAX))
     CFLAGS += -DDTLS_HANDSHAKE_MAX=$(HANDSHAKE_MAX)
 endif
+
+ifneq (,$(filter gcoap,$(USEMODULE)))
+    # Configuring the buffer large enough that a full Gcoap packet can be
+    # encrypted or decrypted.
+
+    # This is the default in gcoap.h, which we don't have access to, so it is copied over.
+    CONFIG_GCOAP_PDU_BUF_SIZE := $(or $(CONFIG_GCOAP_PDU_BUF_SIZE),128)
```

Comment:
How does the `$(or $(CONFIG_GCOAP_PDU_BUF_SIZE),128)` work?
Is this something like

    CONFIG_GCOAP_PDU_BUF_SIZE ?= 128